jerry-core/jmem/jmem-allocator-internal.h

```diff
@@ -24,26 +24,26 @@
  * @{
  */
 
- /**
-  * @{
-  * Valgrind-related options and headers
-  */
- #ifdef JERRY_VALGRIND
- # include "memcheck.h"
+/**
+* @{
```

Comment:
No space before `*`? That is strange for me.

Reply:
Oops, that one was accidental. I'll fix it.